src/components/side-panel/panel-header.js

```diff
@@ -123,17 +122,10 @@ const StyledPanelDropdown = styled.div`
 export const PanelAction = ({item, onClick}) => (
   <StyledPanelAction className="side-panel__panel-header__action"
     data-tip data-for={`${item.id}-action`} onClick={onClick}>
+    <p style={{fontWeight: 'bold'}}>Share</p>
```

Comment:
can you delete the inline style? we try to have all the css style in the `styled.div`declaration above

Reply:
@heshan0131 Deleted. Sorry for this problem, I saw in the project another places with inline style. Sorry again. I commit the changes. If you want to send me a contribution guideline, I will happy to include on PR #108 to avoid this kind of issues.